tests/sources/xml/test_ead.py

```diff
@@ -1144,11 +1144,7 @@ def test_get_notes_transforms_correctly_with_multiple_kinds():
             <bioghist>
                 <head>Biographical Note</head>
                 <p>
-            """
-            "Charles J. Connick (1875-1945) was an American "
-            "<emph>stained</emph> glass artist whose work may be found "
-            "in cities all across the United States."
-            """
+                    Charles J. Connick was an American <emph>stained</emph> glass artist.
```

Comment:
Good call on the `<emph>`!  As we chatted about in our call, no real hard and fast rules about what to include, but the more we mimick stuff we might see, the better.  Dig it.